src/SDK/Language/Dart.php

```diff
@@ -99,7 +99,7 @@ public function getTypeName($type)
                 return 'String';
             break;
             case self::TYPE_FILE:
-                return 'File';
+                return '';
```

Comment:
We need to figure out a way to handle file uploads. This is usually a harder task in most language where we had to implement it.

Reply:
I removed this, because `File` class is for representing file on file system. It's just a reference, not binary content of the file - which I think is needed in this case. Currently there is no type, so it's "dynamic" and user can pass whatever he wants to. So until we'll figure it out - this is the best solution.

Reply:
👌🏻